### ISSUE TITLE
NTR: Increased fecal coproporphyrin 1

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#5684]
**Preferred term label:**  
Increased fecal coproporphyrin 1

**Synonyms**
Elevated fecal coproporphyrin 1; High fecal coproporphyrin 1; Elevated stool coproporphyrin 1; High stool coproporphyrin 1

**Definition (free text, please give PubMed ID)**
Abnormally high concentration of coproporphyrin 3 in feces 

[PMID:3327428](http://www.ncbi.nlm.nih.gov/pubmed/3327428)

**Parent term (use hpo.jax.org/app)**
See issue #5627 - Increased fetal porphyrin

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- [177000](http://omim.org/entry/177000) Protoporphyria, Erythropoietic  
- [176200](http://omim.org/entry/176200) Variegate porphyria  
- [121300](http://omim.org/entry/121300) Hereditary coproporphyria  
- [176000](http://omim.org/entry/176000) Acute intermittent porphyria  

[PMID:10905759](http://www.ncbi.nlm.nih.gov/pubmed/10905759)
[PMID:13765301](https://www.ncbi.nlm.nih.gov/pubmed/13765301)

**Your nano-attribution (ORCID)**  
https://orcid.org/0000-0002-8169-9049
